crates/bevy_graph/src/error.rs

```diff
@@ -0,0 +1,12 @@
+use crate::graphs::keys::{EdgeIdx, NodeIdx};
+
+#[derive(Debug)]
+pub enum GraphError {
+    NodeIdxDoesntExist(NodeIdx),
+    EdgeIdxDoesntExist(EdgeIdx),
+    EdgeBetweenDoesntExist(NodeIdx, NodeIdx),
+    EdgeBetweenAlreadyExists(NodeIdx, NodeIdx),
+    EdgeBetweenSameNode(NodeIdx),
+}
+
+pub type GraphResult<T> = Result<T, GraphError>;
```

Comment:
Remove please: this type alias obfuscates without any real value.

Reply:
So should i use Result<T, GraphError> everywhere directly?